### PR TITLE
Update MDIF import comment handling & add formatting for NF data in Network.write_touchstone

### PR DESCRIPTION
**Pull request by wsapia**

### CODE REVIEW

File: skrf/network.py [hunk 1]
```diff
@@ -2337,6 +2340,26 @@ def write_touchstone(self, filename: str | Path = None, dir: str | Path = None,
...
+            https://docs.python.org/3/library/string.html#format-string-syntax
+            This specifies the formatting in the resulting touchstone file for the GammaOpt magnitude.
+        format_spec_g_opt_phase : string, optional
+            Any valid format specifying string as given by
+            https://docs.python.org/3/library/string.html#format-string-syntax
+            This specifies the formatting in the resulting touchstone file for the GammaOpt phase.
+        format_spec_rn : string, optional
+            Any valid format specifying string as given by
+            https://docs.python.org/3/library/string.html#format-string-syntax
+            This specifies the formatting in the resulting touchstone file for the noise resistance.
```
Comment (jhillairet) [inline skrf/network.py:2362]: I suggest to add various examples in an Example docstring section (to add)

**Comment by wsapia** [comment]
The formatting arguments are all identical to the ones that were originally there, and I think it's common enough that people will know how to use it. That said, I added it to the docs for people who want to see how they're used, hopefully it's clear.

**Review by jhillairet — APPROVED** [review]
Very good idea to add this doc about exporting Touchstone files, thank you!

**Comment by jhillairet** [comment]
Please fix the conflict then it will be merged.

**Comment by wsapia** [comment]
Really not sure why Ruff is so unhappy with the newline at the end of the ipynb file. I can't seem to get it to pass

**Review by jhillairet — APPROVED** [review]

**Comment by jhillairet** [comment]
Thank you for your work. I'am merging it.

**Pull request merged**